coffea/processor/__init__.py

```diff
@@ -38,11 +38,11 @@
     "LazyDataFrame",
     "Weights",
     "PackedSelection",
-    "iterative_executor",
-    "futures_executor",
-    "dask_executor",
-    "parsl_executor",
-    "work_queue_executor",
+    "IterativeExecutor",
+    "FuturesExecutor",
+    "DaskExecutor",
+    "ParslExecutor",
+    "WorkQueueExecutor",
```

Comment:
I would prefer to drop the capitalization for the executor types, because if we make a small change to how the executor is initialized, we can make it so that no user changes are required.